config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -47,6 +47,12 @@
               files="src[\\/]test[\\/]resources[\\/]*"/>
     <suppress checks="RegexpSingleline"
               files="src[\\/]test[\\/]resources-noncompilable[\\/]*"/>
+    <!-- content is generated -->
+    <suppress checks="RegexpSingleline"
+              files="src[\\/]xdocs[\\/]releasenotes_old.xml"/>
+    <suppress checks="RegexpSingleline"
+              files="src[\\/]xdocs[\\/]releasenotes.xml"/>
```

Comment:
Hasn't this become a duplicate of https://github.com/checkstyle/checkstyle/issues/5291 ? can the other issue be closed?

Reply:
Also do we need the same check in the main configuration file?

Reply:
we will take a look at the end of update.
Looks like xdoc still have suppressions for main config.